lang/en/permission.php

```diff
@@ -5,31 +5,39 @@
 return [
     'role' => [
         Role::ROOT => 'Root',
-        Role::STAFF => __('Staff'),
         Role::ADMINISTRATOR => __('Administrator'),
+
+        // moderation & platform staff roles
+
         Role::MODERATOR => __('Moderator'),
-        Role::BETA => __('Beta'),
+        Role::RELEASE_MANAGER => __('Release Manager'),
+        Role::HUB_MANAGER => __('Hub Manager'),
 
-        Role::DEVELOPER => __('Development Access'),
-        Role::DEVELOPER_LEVEL_1 => __('Sr. Developer'),
-        Role::DEVELOPER_LEVEL_2 => __('Developer'),
-        Role::DEVELOPER_LEVEL_3 => __('Jr. Developer'),
-        Role::DEVELOPER_VETERAN => __('Veteran'),
+        // creator roles
 
+        Role::DEVELOPER_STAFF => __('Staff Developer'),
+        Role::DEVELOPER => __('Developer'),
+        Role::DEVELOPER_JUNIOR => __('Junior Developer'),
         Role::ARTIST => __('Artist'),
+        Role::WRITER => __('Writer'),
+        Role::TESTER => __('Tester'),
+
+        // community staff roles
 
         Role::FORUM_MANAGER => __('Forum Manager'),
-        Role::HUB_MANAGER => __('Hub Manager'),
         Role::TICKET_MANAGER => __('Ticket Manager'),
-        Role::RELEASE_MANAGER => __('Release Manager'),
-
         Role::NEWS_MANAGER => __('News Manager'),
-        // Role::NEWS_EDITOR => __('News Editor'),
-
         Role::EVENT_MANAGER => __('Event Manager'),
 
+        // vanity roles assigned by root
+
+        Role::FOUNDER => __('Founder'),
         Role::ARCHITECT => __('Architect'),
         Role::ENGINEER => __('Engineer'),
-        Role::FOUNDER => __('Founder'),
+        Role::BETA => __('Beta'),
+
+        // vanity roles assigned by admins
+
+        Role::DEVELOPER_VETERAN => __('Veteran'),
```

Comment:
Should this label be "Veteran Developer"?

Reply:
good call, updated